### ISSUE TITLE
deREferencing IDA Pro plugin not compatible with IDA 9.0

### ISSUE THREAD

**Issue by mr-tz** [mandiant/VM-Packages#1151]
### What's the problem?

```
Failed while executing plugin_t.init():
Traceback (most recent call last):
  File "C:/Users/flare/AppData/Roaming/Hex-Rays/IDA Pro/plugins/dereferencing.py", line 132, in init
    if not dbg.supported_cpu():
  File "C:\Users/flare/AppData/Roaming/Hex-Rays/IDA Pro/plugins\dereferencing\dbg.py", line 66, in supported_cpu
    info = idaapi.get_inf_structure()
AttributeError: module 'idaapi' has no attribute 'get_inf_structure'
```

https://github.com/danigargu/deREferencing/pull/25 fixes the issue

### Steps to Reproduce

Install and use IDA 9.0 with current deREferencing plugin

### Environment

any?!

### Detected by test suite

No

**Comment by Ana06** [comment]
I have noticed the error as well. The package should be easy to update after [deREferencing](https://github.com/danigargu/deREferencing) has been fixed. If it takes long to fix we can remove the package from the default configuration until it is fixed.

To uninstall the package (and remove deREferencing and consequently the error when opening a file with IDA 9.0): `choco uninstall ida.plugin.dereferencing.vm`. The uinstallation does not need internet.
